Dockerfile

```diff
@@ -3,4 +3,5 @@ FROM python:2.7-onbuild
 RUN apt-get update \
     && apt-get install -y python-protobuf
```

Comment:
1. It works without it. 
2. It's in requirements.txt. Probably error is on some platforms without it's installation.